functional-tests/src/test/java/org/infinispan/online/service/sharedmemory/SharedMemoryServiceTest.java

```diff
@@ -45,6 +49,16 @@ public void should_default_cache_be_accessible_via_hot_rod() throws IOException
       hotRodTester.testBasicEndpointCapabilities(hotRodService);
    }
 
+   @Test(expected = HotRodClientException.class)
+   public void should_default_cache_be_protected_via_hot_rod() throws IOException {
+      hotRodTester.testBasicEndpointCapabilities(hotRodService, false);
```

Comment:
If you agree with me in my previous comment (in `CachingServiceTest`), those bits needs to be updated.